src/ol/render/canvas/canvasreplay.js

```diff
@@ -1376,6 +1402,16 @@ ol.render.canvas.PolygonReplay.prototype.getBufferedMaxExtent = function() {
 
 
 /**
+ * @param {boolean} rightHanded Whether polygons are assumed to follow
+ *     the right-hand rule.
+ */
+ol.render.canvas.PolygonReplay.prototype.setRightHanded =
```

Comment:
There is no way to keep this out of the renderer code? The overhead of "transforming" them at the feature / geom level is too much or some other reason? I.e. always have our geometries be non right-handed internally?